### DISCUSSION TITLE
C-port of stable-diffusion.cpp with more user-friendly code and efficient RAM management

### DISCUSSION THREAD

**Discussion by JohnClaw** [leejet/stable-diffusion.cpp#420] [General]
https://github.com/aagdev/mlimgsynth

**Comment by ganelonhb** [comment]
Cool! Does it currently/plan-to have control net support? With control net support, I can use your code to target lower-end machines for a project I have in mind that I've been building using stable-diffusion.cpp.

**Reply by JohnClaw** [reply]
> Cool! Does it currently/plan-to have control net support? With control net support, I can use your code to target lower-end machines for a project I have in mind that I've been building using stable-diffusion.cpp.

That's not my code. App is being developed by https://github.com/aagdev
